### PR TITLE
create table vets

### PR DESCRIPTION
**Pull request by Regiss05**
This PR should:

**Create a table named vets**
Screen Shot 2022-07-09 at 21 04 59

**Create specialization table**
Screen Shot 2022-07-09 at 21 17 06

**create visits table**
Screen Shot 2022-07-09 at 21 22 37

**Insert the following data for vets:**
> Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
> Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
> Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
> Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
Screen Shot 2022-07-09 at 21 41 51

**Insert the following data for specialties:**
> Vet William Tatcher is specialized in Pokemon.
> Vet Stephanie Mendez is specialized in Digimon and Pokemon.
> Vet Jack Harkness is specialized in Digimon.
Screen Shot 2022-07-09 at 21 44 24

**Insert  data into visits table**
Screen Shot 2022-07-09 at 21 47 37

**Who was the last animal seen by William Tatcher**
Screen Shot 2022-07-09 at 21 55 13

**How many different animals did Stephanie Mendez see**
Screen Shot 2022-07-09 at 21 58 59

**List all vets and their specialties, including vets with no specialties.**
Screen Shot 2022-07-09 at 22 01 51

**all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.**
Screen Shot 2022-07-09 at 22 05 04

**What animal has the most visits to vets**
Screen Shot 2022-07-09 at 22 07 42

**Who was Maisy Smith's first visit?**
Screen Shot 2022-07-09 at 22 09 34

**How many visits were with a vet that did not specialize in that animal's species?**
Screen Shot 2022-07-09 at 22 13 44

**What specialty should Maisy Smith consider getting? Look for the species she gets the most.**
Screen Shot 2022-07-09 at 22 15 52

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -109,4 +109,64 @@ COUNT(animals.owner_id) as Animal_Highest_Count
...
+WHERE vets.name ='Maisy Smith' ORDER BY visits.date_of_visit ASC LIMIT 1;
+
+-- SELECT date_of_visit as Most_Recent_visit, 
+animals.date_of_birth as animal_dob, animals.escape_attempts,
+animals.neutered as neutered_status, animals.weight_kg as animal_weight,
+vets.name as vet_name, vets.age as vet_age, vets.date_of_graduation
+FROM visits JOIN animals ON animals.id = visits.animals_id
+JOIN vets ON vets.id = visits.vets_id
+ORDER BY date_of_visit DESC
+LIMIT 1;
```
Comment (sja-thedude) [inline queries.sql:154]: - [ ] Kindly add a screenshot for the query "Details for most recent visit: animal information, vet information, and date of visit."

File: schema.sql [hunk 2]
```diff
@@ -37,3 +37,57 @@ ALTER TABLE animals ADD owner_id INT
...
+       ((SELECT id FROM animals WHERE name = 'Pikachu'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), 'May-14-2020'),
+       ((SELECT id FROM animals WHERE name = 'Devimon'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), 'May-04-2021'),
+       ((SELECT id FROM animals WHERE name = 'Charmander'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), 'Feb-24-2021'),
+       ((SELECT id FROM animals WHERE name = 'Plantmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), 'Dec-21-2019'),
+       ((SELECT id FROM animals WHERE name = 'Plantmon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), 'Aug-10-2020'),
+       ((SELECT id FROM animals WHERE name = 'Plantmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), 'Apr-07-2021'),
+       ((SELECT id FROM animals WHERE name = 'Squirtle'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), 'Sep-29-2019'),
+       ((SELECT id FROM animals WHERE name = 'Angemon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), 'Oct-03-2020'),
+       ((SELECT id FROM animals WHERE name = 'Angemon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), 'Nov-04-2020'),
```
Comment (sja-thedude) [inline schema.sql:87]: - [ ] Kindly add Everything that populates tables with data should be saved in the data.sql file.

- [ ] Also make sure to add vets table data.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Regiss05, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional readme
✔️ Correct gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 2 above)
Comment (Whoistolu) [reply schema.sql:87]: - [ ] [REQUIRED] Kindly ensure that you add data for the `vets` table as recommended by the previous reviewer. You can do that in the `data.sql` file. This is a core requirement for this milestone.

File: queries.sql (hunk 1 above)
Comment (Whoistolu) [reply queries.sql:154]: Resolved 👍🏽

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Regiss05 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @Regiss05, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**